app/models/condition.rb

```diff
@@ -293,4 +297,42 @@ def self.import_from_hash(condition, options={})
 
     return c, status
   end
+
```

Comment:
Ah, can you add the following to the description of this PR and the commit that adds some of the code back?
A link to `#3708` and a short description of why it was removed in that PR and why it's ok to put it back?